.travis.yml

```diff
@@ -15,6 +15,7 @@ install:
 - pip install .
 - pip install -r requirements.txt
 - pip install -r test-requirements.txt
+- pip freeze
```

Comment:
I would leave it for future inspections . in my opinion it improves the travis log.